ng-deploy/src/builders/actions/deploy.ts

```diff
@@ -1,8 +1,40 @@
-import { execSync, spawnSync } from "child_process";
+import { BuilderContext } from '@angular-devkit/architect/src/index2';
+import { join } from 'path';
 
-export default function deploy() {
-    const bin = execSync('(cd ' + __dirname + ' && npm bin)').toString().trim();
-    const firebaseBin = bin + '/firebase';
-    spawnSync(firebaseBin, ['login'], {stdio: 'inherit'});
-    spawnSync(firebaseBin, ['deploy', '--only', 'hosting'], {stdio: 'inherit'});
+const tools = require('firebase-tools');
```

Comment:
Hm, were you able to get proper output from this thing? 
I switched to running CLI commands because it was showing everything in place one line at a time

Reply:
I didn't hit any issues. Using macOS with tmux.